development.ini

```diff
@@ -9,6 +9,8 @@ use = egg:pyoes
 pyoes.banner = Dit is een speciale boodschap!
 pyoes.banner.detail = En dit is de langere boodschap...
 
+cookie_banner = true
```

Comment:
ik heb de cookie banner opgezet in de demo pagina, maar er komen nog errors, check jij eens? hij vindt de build dir precies niet van cokkieconsent.